projects/admin/src/app/circulation/circulation-routing.module.ts

```diff
@@ -58,6 +59,10 @@ const routes: Routes = [
         path: 'pending',
         component: PendingComponent,
         canActivate: [ PermissionGuard ], data: { permissions: [ PERMISSIONS.CIRC_ADMIN ] }
+      },{
+        path: 'ill',
+        component: IllRequestComponent,
+        canActivate: [ PermissionGuard ], data: { permissions: [ PERMISSIONS.CIRC_ADMIN ] }
```

Comment:
Maybe add the permissions "ILL_SEARCH" ?

Reply:
All routes have the `PERMISSIONS.CIRC_ADMIN` permission. Is it different for the ILL route ?